### PR TITLE
Task #199542 style: update dashboard counts with given values

### PR DESCRIPTION
**Pull request by shreyas1434shinde**
code give by shivani

### CODE REVIEW

File: tenant/sunbird/learnathon.html [hunk 1]
```diff
@@ -159,25 +159,22 @@ <h4 class="abt-sec-title title-top-margin">What is Learnathon?</h4>
 
   <div class="total-section">
     <div>
-      <div id="total" class="number">363</div>
+      <div id="total" class="number">18647</div>
```
Comment (komalm) [inline tenant/sunbird/learnathon.html:162]: @shreyas1434shinde Please Update MR Title as per std format

File: tenant/sunbird/index.html [hunk 2]
```diff
@@ -128,7 +128,8 @@ <h1 class="sb-text-upper mt-0">Learning Platform</h1>
                             <h2 class="sb-text-upper">National Urban</h2>
                             <h1 class="sb-text-upper mt-0">Learning Platform</h1>
                             <p class="sb-font">Launching SBM 2.0 with the vision to make ‘Garbage Free Cities’, <br> which aims to sustain and advance the progress in urban <br> sanitation and waste management</p>
-                            <div><a class="sb-btn-default mt-10" href="https://nulp.niua.org/resources?SBMRegistration">Sign In</a>
+                            <div>
```
Comment (komalm) [inline tenant/sunbird/index.html:131]: @shreyas1434shinde  I'll not be going to merge this command til the title is not in the proper format

File: tenant/sunbird/index.html [hunk 3]
```diff
@@ -146,7 +147,8 @@ <h1 class="sb-text-upper mt-0">Learning Platform</h1>
                             <h2 class="sb-text-upper">National Urban</h2>
                             <h1 class="sb-text-upper mt-0">Learning Platform</h1>
                             <p class="sb-font">एसबीएम 2.0 ‘कचरा मुक्त शहर ’ बनाने की उद्देश्य के साथ लॉन्च किया जा रहा है, <br> जिसका लक्ष्य शहरी स्वच्छता और अपशिष्ट प्रबंधन में प्रगति को बनाए रखना और <br> आगे बढ़ाना है</p>
-                            <div><a class="sb-btn-default mt-10" href="https://nulp.niua.org/resources?SBMRegistration">Sign In</a>
+                            <div>
+                                <!-- <a class="sb-btn-default mt-10" href="https://nulp.niua.org/resources?SBMRegistration">Sign In</a> -->
```
Comment (komalm) [inline tenant/sunbird/index.html:151]: @shreyas1434shinde Do not remove, show that button in disable format

Comment (shreyas1434shinde) [reply tenant/sunbird/index.html:151]: @komalm okay

File: tenant/sunbird/learnathon.html (hunk 1 above)
Comment (shreyas1434shinde) [reply tenant/sunbird/learnathon.html:162]: @komalm updated

File: tenant/sunbird/index.html (hunk 2 above)
Comment (shreyas1434shinde) [reply tenant/sunbird/index.html:131]: @komalm updated

**Pull request merged**